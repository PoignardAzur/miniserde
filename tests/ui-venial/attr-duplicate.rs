use miniserde_derive_venial::Serialize;

// FIXME - handle attributes with multiple items

#[derive(Serialize)]
struct Struct {
    #[serde(rename = "A")]
    #[serde(rename = "B")]
    x: i32,
}

fn main() {}
