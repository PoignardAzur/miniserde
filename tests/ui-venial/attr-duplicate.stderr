error: duplicate rename attribute
 --> tests/ui-venial/attr-duplicate.rs:8:5
  |
8 |     #[serde(rename = "B")]
  |     ^^^^^^^^^^^^^^^^^^^^^^
